crates/client-api-messages/src/websocket.rs

```diff
@@ -222,6 +275,99 @@ pub enum ServerMessage<F: WebsocketFormat> {
     IdentityToken(IdentityToken),
     /// Return results to a one off SQL query.
     OneOffQueryResponse(OneOffQueryResponse<F>),
+    /// Sent in response to a `SubscribeSingle` message. This contains the initial matching rows.
+    SubscribeApplied(SubscribeApplied<F>),
+    /// Sent in response to an `Unsubscribe` message. This contains the matching rows.
+    UnsubscribeApplied(UnsubscribeApplied<F>),
+    /// Communicate an error in the subscription lifecycle.
+    SubscriptionError(SubscriptionError),
+    /// Informs of changes to subscribed rows.
+    SubscriptionUpdate(SubscriptionUpdate<F>),
```

Comment:
Where did this message come from? Isn't this the same thing as `TransactionUpdate`?

Reply:
Wait, apparently it's `InitialSubscription` in response to `Subscribe`. But again, where did this come from?

Reply:
Looks like I copied this unused message over from https://github.com/clockworklabs/SpacetimeDB/pull/1997